templates/base.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE HTML>
+<html lang="en">
+<head>
+    {% block title %}

```

Comment:
using 2 tablespaces is the best practice
https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD

Reply:
please, change everywhere